packages/cf-builder-pagination/example/basic/component.js

```diff
@@ -1,6 +1,5 @@
-const React = require('react');
-const { render } = require('react-dom');
-const { PaginationBuilder } = require('../../src/index');
+import React from 'react';
```

Comment:
Changing the module syntax for our imports doesn't represent a SEMVER incompatible API change, as far as I'm aware.